From 2f7c1fd23d9faad5bcf8cdfe04c1632352bc0136 Mon Sep 17 00:00:00 2001
Message-Id: <2f7c1fd23d9faad5bcf8cdfe04c1632352bc0136.1640138222.git.geliang.tang@suse.com>
From: Tom Rix <trix@redhat.com>
Date: Wed, 23 Dec 2020 11:44:25 -0800
Subject: [PATCH] igb: remove h from printk format specifier
Git-commit: 2f7c1fd23d9faad5bcf8cdfe04c1632352bc0136
Patch-mainline: v5.12-rc2
References: bsn#47


This change fixes the checkpatch warning described in this
commit cbacb5ab0aa0 ("docs: printk-formats: Stop encouraging use of
unnecessary %h[xudi] and %hh[xudi]")

Standard integer promotion is already done and %hx and %hhx is useless
so do not encourage the use of %hh[xudi] or %h[xudi].

Signed-off-by: Tom Rix <trix@redhat.com>
Signed-off-by: Tony Nguyen <anthony.l.nguyen@intel.com>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/net/ethernet/intel/igb/igb_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/intel/igb/igb_main.c b/drivers/net/ethernet/intel/igb/igb_main.c
index 23e50de94474..bd0594ac3ae7 100644
--- a/drivers/net/ethernet/intel/igb/igb_main.c
+++ b/drivers/net/ethernet/intel/igb/igb_main.c
@@ -3156,7 +3156,7 @@ static int igb_probe(struct pci_dev *pdev, const struct pci_device_id *ent)
 	 * the PCIe SR-IOV capability.
 	 */
 	if (pdev->is_virtfn) {
-		WARN(1, KERN_ERR "%s (%hx:%hx) should not be a VF!\n",
+		WARN(1, KERN_ERR "%s (%x:%x) should not be a VF!\n",
 			pci_name(pdev), pdev->vendor, pdev->device);
 		return -EINVAL;
 	}
-- 
2.31.1

